### PR TITLE
Add VoyageDB to contributor list

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Adds VoyageDB for their multiple contributions: https://github.com/R2Northstar/NorthstarMods/commits?author=DBmaoha

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
@DBmaoha thoughts? Just to double check that you're fine with getting added from your side ^^

**Comment by DBmaoha** [comment]
Thanks very much! also really appriciate reviewing my pull requests. I'm making a pr about bounty hunt fix, will try to do better formatting by myself

File: dist/script/credits.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ addContributor("core", "https://github.com/p0358", "https://avatars.githubuserco
 addContributor("contrib", "https://github.com/BigSpice", "https://avatars.githubusercontent.com/u/23240514?v=4", "juicy", "VTOL, Skins")
 addContributor("contrib", "https://github.com/taskinoz", "https://avatars.githubusercontent.com/u/7439692?v=4", "taskinoz", "Moderator, NavMeshes")
 addContributor("contrib", "https://github.com/laundmo", "https://avatars.githubusercontent.com/u/24855949?v=4", "laundmo", "Modding Docs, Chathooks")
+addContributor("contrib", "https://github.com/DBmaoha/", "https://avatars.githubusercontent.com/u/56738369?v=4", "VoyageDB", "Frontier War, Bounty Hut, Squirrel bug fixes")
```
Comment (pg9182) [inline dist/script/credits.js:32]: Bounty *Hut*...

Comment (GeckoEidechse) [reply dist/script/credits.js:32]: Whoops, fixed in e2c3ec969704d82d20ee42a7659deadbf0de153f

**Pull request merged**